docs/stable/1-client.md

```diff
@@ -165,15 +167,19 @@ It uses Ethereum Node Records (ENR) to report discovered peers.
 Both peers discovery mechanisms use topics to provide peers with certain capabilities.
 There is no point in returning peers that do not support a particular protocol.
 Status nodes that want to be discovered MUST register to Discovery V5 and/or Rendezvous
-with the `whisper` topic. Status nodes that are mail servers and want to
+with the `whisper` and/or `waku` topic(s). Status nodes that are `Mailservers` and want to
```

Comment:
Did we definitely change this to `waku` topic?

Reply:
I don't know. It is related to one of my outstanding issues. There is a reference in the specs to `whispermail`, but I haven't been able to find if it was changed to `wakumail` with Waku. 

In this section I presumed that Waku would have its `waku` topic set.

Reply:
Don't know. I'd grep the status-go code for this. If you can only find whispermail we didn't change it.

Reply:
https://github.com/status-im/status-go/search?q=whispermail&unscoped_q=whispermail and https://github.com/status-im/status-go/search?q=wakumail&unscoped_q=wakumail seems like no

Reply:
That is a useful feature! Ok, I suppose that answers the "do we" question, the next question would be "should we?".

I'll remove the changes I've made here and I'll also break this into a separate issue.